app/components/Inspector.js

```diff
@@ -1,13 +1,32 @@
-import React, { Component, PropTypes } from 'react';
-import styles from './Session.css';
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import { Card } from 'antd';
+import Screenshot from './Inspector/Screenshot';
+import SelectedElement from './Inspector/SelectedElement';
+import Source from './Inspector/Source';
+import InspectorStyles from './Inspector.css';
 
 export default class Inspector extends Component {
 
+  componentWillMount () {
+    this.props.applyClientMethod({methodName: 'source'});
+    this.props.bindAppium();
```

Comment:
could there ever be a race condition in which we haven't bound the listener even though the source command might have tried to trigger the event?